bindings/bind_py_representation_calculator.cc

```diff
@@ -40,15 +40,12 @@ namespace rascal {
 
     py::class_<Calculator, CalculatorBase> representation(
         mod, representation_name.c_str());
-    // use custom constructor to pass json formated string as initializer
-    // an alternative would be to convert python dict to json internally
-    // but needs some work on in the pybind machinery
-    representation.def(py::init([](std::string & hyper_str) {
+    representation.def(py::init([](const py::dict & hyper) {
       // convert to json
-      json hypers = json::parse(hyper_str);
+      json hypers = hyper;
```

Comment:
There is a 'bad' interaction with the json construction and initialiazer list. Since an initialiazer list is a list the resulting json object is always a json::array of what you put in the braces but here we want to convert the py::dict to a json::object.